scripts/zones/Behemoths_Dominion/mobs/King_Behemoth.lua

```diff
@@ -23,7 +23,14 @@ end;
 function onMobDeath(mob, killer)
 
     killer:addTitle(BEHEMOTH_DETHRONER);
-
+	
+	if(math.random((1),(100)) <= 5) then -- Hardcoded "this or this item" drop rate until implemented.
+		SetDropRate(1936,13566,1000); -- Defending Ring
+		SetDropRate(1936,13415,0);
+    else
+		SetDropRate(1936,13566,0);

```

Comment:
Just so you know, mixing tabs and spaces makes display look unaligned even when its technically correct. That's why github has it looking different than in your editor.
![isMagic](http://s14.postimg.org/nqrwrconl/npp_settings.png)

(you don't really have to change anything, just letting you know about helpfull features here)